bokehjs/src/coffee/models/annotations/color_bar.coffee

```diff
@@ -203,7 +203,8 @@ export class ColorBarView extends AnnotationView
     image = @model._computed_image_dimensions()
     [x_offset, y_offset] = [image.width * nx, image.height * ny]
 
-    [sx, sy] = @model._tick_coordinates().major
+    tick_coords = @model._tick_coordinates()
```

Comment:
`@model._tick_coordinates()` looks to be called at least three times. I think this could be improved:
* rename `_tick_coordinates` to `tick_info` (no underscore, update name to reflect changes)
* call `tick_info()` once,  before `_draw_major_ticks`, etc, and pass the info to these methods instead of having them each call `tick_info()` separetely, e.g. `@_draw_major_ticks(ctx, tick_info)`